README.adoc

```diff
@@ -6,7 +6,7 @@
 This site contains the API and Extension registries for the OpenGL family
 APIs - OpenGL, OpenGL ES, and OpenGL SC. It includes API specifications;
 specifications of Khronos- and vendor-approved extensions; header files
-corresponding to the specifications; the XML API Registry definining each
```

Comment:
Fixing the typo is fine, but I don't believe the other two changes are appropriate.